src/IntegratesQueryBuilder.js

```diff
@@ -0,0 +1,43 @@
+import { Mixin } from 'mixwith'
+import Javel from '@/Javel'
```

Comment:
I'd rather register the modules a little bit differently.

Instead of exposing a global `Javel` class...

```js
import { Javel } from 'javel'

Javel.registerOptionalModule(name, dependency)
let dependency = Javel.resolveOptionalModule(name)
```

... I'd rather expose the functions directly

```js
import { registerModule, resolveModule } from 'javel'

registerModule(name, dependency)
let dependency = resolveModule(name)
```

That way we can even rename the `Javel` class into something more specific like `ModuleRegistration`.

What do you think?

Reply:
I like your idea.